pkg/generators/readers.go

```diff
@@ -237,6 +237,47 @@ func (g *ReadersGenerator) generateHelpers() error {
 			}
 			return
 		}
+
+		// ParseInteger reads a string and parses it to integer,
+		// if an error ocurred it returns a non-nil error.
+		func ParseInteger(s string) (*int, error) {
```

Comment:
I think these functions should receive the complete query object and the name of the parameter. That way you can generate useful error messages l like `value 'abc' isn't valid for the 'size' parameter because it isn't an integer`.